docs/tutorial.md

```diff
@@ -1,8 +1,13 @@
 
 # Create your own activity
 
-With Sugarizer, it's easy to create your own activity with a bunch of HTML and JavaScript.
+<<<<<<< HEAD
```

Comment:
Something goes wrong here.

Reply:
Actually I did some updations in the master branch accidentally. So, Let me close this PR and create another one.